bin/controller.ts

```diff
@@ -28,9 +28,20 @@ const job1: Job = new NormalJob('first', new Set([job4, job2]))
 const job5: Job = new NormalJob('fifth', new Set([job1]))
 
 const client: Client = new Http2Client(new HeapJobOrderer([job5]))
-client.on('progress', console.log)
+client.on('progress', (FinishedJob) => {
+	console.log(`${FinishedJob.getName()} completed\n`)
+})
 once(client, 'done').then(clientDone).catch(console.error)
 
+process.on('SIGINT', () => {
+	console.log('\nClosing Daemons\n')
+
+	client.closeAllDaemonsAndFinish()
+
+	console.log('Exit Process\n')
+	process.exit()
```

Comment:
It shouldn't be necessary to force-exit the process. Ideally, once you close all the sockets, you just wait for the event loop will run out. What happens if you remove this line?

Reply:
Yup do not need to force-exit. Removed this line.